config/blacklight.yml

```diff
@@ -1,12 +1,28 @@
 development:
   adapter: solr
-  url: <%= "#{Settings.solr.url}/solr/#{Settings.solr.collection}" %>
+  <% if Settings.solr.password %>
```

Comment:
What about just `SolrConfig.new.url` here and it figures it out for us. We can also copy what we did in Scholarsphere, if that works: https://github.com/psu-libraries/scholarsphere/blob/develop/config/application.rb#L48

Reply:
`SolrConfig` isn't available here, not sure why.

Reply:
Try putting `require 'psulib_blacklight/solr_config'` in `application.rb` but inside the Application class definition, like here https://github.com/psu-libraries/scholarsphere/blob/62d9613d503b29ffef05e113b8a032bc6943c793/config/application.rb#L27

Reply:
I've added a `query_url` method for this use case, and simplified the blacklight config. i hope it suffices